control-plane/subcommand/inject-connect/command.go

```diff
@@ -223,6 +224,9 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagLogLevel, "log-level", zapcore.InfoLevel.String(),
 		fmt.Sprintf("Log verbosity level. Supported values (in order of detail) are "+
 			"%q, %q, %q, and %q.", zapcore.DebugLevel.String(), zapcore.InfoLevel.String(), zapcore.WarnLevel.String(), zapcore.ErrorLevel.String()))
+	c.flagSet.StringVar(&c.flagLogLevelDataplane, "log-level-dataplane", zapcore.InfoLevel.String(),
```

Comment:
I guess the default logLevel here should be an empty string to maintain backward compatibility.